### DISCUSSION TITLE
Check out beautiful audio player, inspired by WaveSurfer.js and Dribbble!

### DISCUSSION THREAD

**Discussion by letavocado** [katspaugh/wavesurfer.js#2829] [Show and tell]
![output](https://github.com/katspaugh/wavesurfer.js/assets/31704887/a1addbef-7d64-436b-9f2e-f27ceec2b5c2)

[Screencast from 2023-06-04 01-26-19.webm](https://github.com/katspaugh/wavesurfer.js/assets/31704887/20d17cbb-9c23-40db-82d2-6a799499db5e)

Inspired by [WaveSurfer.js](https://github.com/katspaugh/wavesurfer.js) & [Dribble](https://dribbble.com/shots/9785864--Music-Player-Audio-Scrubbing).

![image_processing20200127-1311-x7n8fw-min](https://github.com/katspaugh/wavesurfer.js/assets/31704887/8ddda191-40ed-4868-bb3a-ae423bbfc699)

**Comment by Manfredt** [comment]
How did you generate the waveform with Perlin noise?

**Comment by letavocado** [comment]
Hey @Manfredt! Sorry for delay. Special for you https://github.com/letavocado/vue-waudio. 

Generating WaveForm in https://github.com/letavocado/vue-waudio/blob/main/src/components/WaveForm.vue

Cheers! ✨

**Comment by letavocado** [comment]
Initial version. I'll maintaining it. Feel free to open issues/pr's. Thanks!
